.github/contributing.md

```diff
@@ -181,7 +181,7 @@ Shortcut for starting the SFC Playground in local dev mode. This provides the fa
 
 ### `nr dev-esm`
 
-Builds and watches `vue/dist/vue-runtime.esm-bundler.js` with all deps inlined using esbuild. This is useful when debugging the ESM build in a reproductions that require real build setups: link `packages/vue` globally, then link it into the project being debugged.
+Builds and watches `vue/dist/vue-runtime.esm-bundler.js` with all deps inlined using esbuild. This is useful when debugging the ESM build in a reproduction that require real build setups: link `packages/vue` globally, then link it into the project being debugged.
```

Comment:
I think there's a mismatch here between `reproduction` and `require`.

It could be written:

> in reproductions that require

or:

> in a reproduction that requires

Reply:
Thanks a lot for pointing that out! I've made the change to "in a reproduction that requires" as you suggested. Really appreciate your help!